common/config/rush/pnpm-lock.yaml

```diff
@@ -8237,6 +8391,23 @@ packages:
       p-event: 5.0.1
     dev: false
 
+  /cpx@1.5.0:
```

Comment:
These have been reintroduced from #5285 and will need removed otherwise will reintroduce fixed dependabot alerts